src/faults/AbstractPerturbation.jl

```diff
@@ -72,7 +77,7 @@ function simulate(np::AbstractPerturbation, powergrid::PowerGrid, x1::Array, tim
     cb2 = DiscreteCallback(((u,t,integrator) -> t in np.tspan_fault[2]), regularState)
```

Comment:
I think we should use [PresetTimeCallbacks](https://diffeq.sciml.ai/stable/features/callback_library/#PresetTimeCallback) instead. They will trigger only once and at the exact same time. Otherwise one needs to make sure that the fault times are actually visited by the solver?

Reply:
@hexaeder Sounds good. Could you make a commit for that?